crates/block-producer/src/mempool/tests.rs

```diff
@@ -16,6 +17,55 @@ impl Mempool {
     }
 }
 
+// OTEL TRACE TESTS
+// ================================================================================================
+
+#[tokio::test]
+#[serial]
```

Comment:
Should we use a specific key for this? If yes, could you add it to the `setup_test_tracing` docs as well.

```suggestion
#[serial(otel)]
```